pkg/scaffold/v2/webhook/webhook.go

```diff
@@ -91,10 +91,14 @@ var (
 package {{ .Resource.Version }}
 
 import (
+{{ if or .Validating .Defaulting }}
```

Comment:
why the ifs are required to be added here?

Reply:
Those packages are probably only being used in case you use those flags. As before this change `goimport` tool was being used, this tool removed them, but as this PR swaps `goimport` for `go fmt` imports are not being checked any more and it would scaffold a file with two import compile errors.

Reply:
Exactly - using gofmt instead of goimports requires us to be precise about what we want to import (good) and not import (less good)